src/Microsoft.NET.Sdk.Functions.Generator/FunctionJsonConverter.cs

```diff
@@ -173,36 +175,78 @@ private bool TryGenerateFunctionJsons()
                 AssemblyResolver = resolver
             };
 
-            var module = ModuleDefinition.ReadModule(_assemblyPath, readerParams);
+            loadContext.Resolving += (context, assemblyName) =>
```

Comment:
This lambda is fairly large -- can you split it out into it's own method for ease-of-reading?